lib/cocoapods/installer.rb

```diff
@@ -114,6 +114,12 @@ def install!
     end
 
     def prepare
+      # Raise if pwd is inside Pods
+      if (Pathname.pwd <=> sandbox.root) >= 0
```

Comment:
Is this guaranteed to work? I'm not sure pathname's comparison operator is that clever